### ISSUE TITLE
Load-test with mainnet block update rate

### ISSUE THREAD

**Issue by edkeyes-apollo** [vessenes/persona#31]
Aylab is creating a new mainnet block every few seconds, so anything in the Persona site that depends on the blockchain update subscription is going to be put under a lot more stress in the real launch.

So it may be worth doing some load testing and an audit of what things trigger a full page refresh, etc.

**Comment by vessenes** [comment]
Yeah, It's a long term goal to run pretty much everything through the server -- we'll see what the vats do to everyone's latency, too..
